lib/types/multipart.js

```diff
@@ -288,7 +287,6 @@ function skipPart (part) {
 }
 
 function FileStream (opts) {
-  if (!(this instanceof FileStream)) { return new FileStream(opts) }
```

Comment:
isn't this helpful?

Reply:
not really. we use this Stream internally and we call it with 'new' operator in line 172. So this line can never be active, and it is not possible to unit test it except we export the function, which makes no sense, as it is not meant to be used externally.